src/components/YourCompany/CompanyName/CorrectCompanyName.vue

```diff
@@ -1,31 +1,33 @@
 <template>
   <v-form id="correct-name-form" ref="correctNameForm" v-model="valid" lazy-validation>
-    <v-layout row>
-      <v-flex>
+    <v-row no-gutters>
+      <v-col>
         <v-text-field
           v-model="companyName"
           id="company-name-input"
+          class="mb-n3"
           filled
-          persistent-hint
           :rules="companyNameRules"
-          data-test="business-identifier"
-        ></v-text-field>
-      </v-flex>
-    </v-layout>
+          @keyup="uppercase('companyName')"
```

Comment:
This updates `this.companyName`.

Reply:
What did (does) persistent-hint do?

Reply:
https://vuetifyjs.com/en/components/text-fields/#hint